test/REXUniswapV3Market.test.ts

```diff
@@ -1515,5 +1515,25 @@ describe('REXUniswapV3Market', () => {
         })
         .exec(bobSigner)
     })
+
+    it('#4.2 Should return the correct next distribution time', async () => {
+      const gasPrice = 320 // 300 GWEI
+      const gasLimit = 120000
+      const inflowRate = 10555 // infow in gwei -> 0.38 usd / hr
+      const lastDistributedAt = 1651261812
```

Comment:
Same here, I think `inflowRate` and `lastDistributedAt` can be gotten internal to the method vs. as an argument to the function.